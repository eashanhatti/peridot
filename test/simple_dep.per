define T : Type = Record {};
define x : T = record {};
