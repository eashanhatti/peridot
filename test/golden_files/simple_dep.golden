((fromList [], fromList [], fromList []),
 Declare
   Obj
   (Rigid (NameObjIntro (Id {unId = 0})))
   (Rigid (TypeType Obj))
   (Define
      (Rigid (NameObjIntro (Id {unId = 0})))
      (RecType (fromList []))
      (Declare
         Obj
         (Rigid (NameObjIntro (Id {unId = 1})))
         (GlobalVar (Id {unId = 0}))
         (Define
            (Rigid (NameObjIntro (Id {unId = 1})))
            (RecIntro (fromList []))
            (Rigid (TypeType Obj))))))
